programs/survival/rope_ladders.sc

```diff
@@ -4,27 +4,88 @@
 // (Carpet Mod 1.4.9)
 //
 // Right clicking on a ladder with an other one, will extend the existing one down.
-///
+
+// 'Updated' by opsaaaaa for 1.19
+// added break unsuported ladder fearue
```

Comment:
I think it ought to say something along the lines of `by BisUmTo and opsaaa`, cos he forgot to add his own name.